common/validations.js

```diff
@@ -0,0 +1,58 @@
+const CONSTANT = require('./constant')
+module.exports = {
+
+    LOGIN:(userData, cb)=>{
```

Comment:
you don't need to check every value, create a function and pass the required value and it will return the missing keys other return all the key's in the JSON object so that we can use that values in the controllers